src/ApplicationLogs/LogReader.cs

```diff
@@ -173,7 +174,7 @@ public bool ShouldThrowExceptionFromCommit(Exception ex)
 
         static string GetExceptionMessage(Exception exception)
         {
-            if (exception == null) return "Engine faulted.";
+            if (exception == null) return null;
```

Comment:
We want `{ "exception": null } ` or `{}`?

Reply:
@Tommo-L said `{ "exception": null }` ✅ `{}` ❌

Reply:
> We want `{ "exception": null } ` or `{}`?

How about:
` if (GetExceptionMessage(appExec.Exception)!= null)
            {
                trigger["exception"] = GetExceptionMessage(appExec.Exception);
            }
`